### PR TITLE
preserve LICENSE from codespeed

### PR DESCRIPTION
**Pull request by dt**

### CODE REVIEW

**Comment by bdarnell** [comment]
Is this repo going to contain a fork of codespeed or a configuration for using codespeed? (that is, why did it start with a partial copy of the repo?) If this is a fork, then I think we should start with a proper fork of the codespeed repo, and if not then I don't think we need the codespeed directory here.

**Comment by dt** [comment]
I initially wanted a config, using it via a req.txt dep, but their latest version on pypi is incompatible with their sample project so I had to vendor it from master (the `codespeed` directory).

Even so, the `roachspeed` dir is a derivative of the upstream's `sample_project` (the setup instructions suggest making a copy and adapting it) -- so we need to retain the lic either way.

**Comment by bdarnell** [comment]
You can depend on a git repo in `req.txt`. The syntax is `git+https://github.com/tobami/codespeed@master#egg=codespeed`

Or could you get the sample project from the tag corresponding to the pypi release? (https://github.com/tobami/codespeed/tree/0.10.1)

**Comment by dt** [comment]
Yeah, my attempt at using the sample project at that tag also ran into some issues. Making this a proper fork that just adds the Docker files and `roach speed` config seems like it might make sense for now.

**Comment by bdarnell** [comment]
I think it would be best to have only the modified `sample_project` in this repo instead of a full fork; i can help with the python installation issues if you have trouble with installing from git.

**Comment by dt** [comment]
Ok, I think we should just replace this repo with this alternate history (4 commits):
https://github.com/dt/roachspeed/commits/alt
TL;DR: a clean import of `sample_project` and license files (1), that is then renamed (2), packaged and configured to depend on upstream `codespeed` (3) and finally slightly customized (4).

**Comment by bdarnell** [comment]
Yup, that branch looks good to me (But s/roachbench/roachspeed/g. Or go the other way and rename the whole thing to roachbench if you'd rather think of a giant cockroach you can sit on instead of a speedy little roach)
